### PR TITLE
Set up an example db/seeds

### PR DESCRIPTION
**Pull request by dkniffin**
## Why?

- #85

## What Changed?

- Added a template seeds.rb
- Added an example seed class

### CODE REVIEW

File: spec/support/shared_examples/basic_setup_steps.rb [hunk 1]
```diff
@@ -119,6 +131,10 @@
     expect(File).to exist("#{project_path}/spec/support/request_helpers.rb")
   end
 
+  it "includes the seeds_spec test" do
+    expect(File).to exist("#{project_path}/spec/seeds/seed_spec.rb")
```
Comment (TylerRockwell) [inline spec/support/shared_examples/basic_setup_steps.rb:135]: Is this `seed_spec` coming from somewhere? I don't see it mentioned anywhere else in this PR.

Comment (dkniffin) [reply spec/support/shared_examples/basic_setup_steps.rb:135]: Yes. It was implemented in a previous PR, so it's already on master: https://github.com/smashingboxes/boxcar/blob/master/templates/seed_spec.rb

**Review by TylerRockwell — APPROVED** [review]
Looks good 👍

**Comment by dkniffin** [comment]
Thanks, @TylerRockwell !

**Pull request merged**